src/sql/workbench/services/tableDesigner/browser/tableDesignerPublishDialog.ts

```diff
@@ -78,8 +80,12 @@ export class TableDesignerPublishDialog extends Modal {
 
 	protected renderBody(container: HTMLElement) {
 		const body = DOM.append(container, DOM.$('.table-designer-publish-dialog'));
-		const markdownElement = this._markdownRenderer.render({ value: this._report }).element;
-		DOM.append(body, markdownElement);
+		if (this._format === 'plaintext') {
+			body.innerText = this._report;
+		} else { // markdown
```

Comment:
Would suggest actually checking for markdown as a value. If the format is unknown it should probably log a warning but default to plaintext.